main.js

```diff
@@ -160,4 +160,127 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+const tvTypes = inventory.map((tv) => {
+  return tv.type
+})
+
+console.log(tvTypes)
+
+const soldOut = inventory.filter((arr) => {
+  if (arr.originalStock === arr.sold) {
+    return true
+  }
+})
+
+console.log(soldOut)
+
+const hasAmbiLight = inventory.filter((arr) => {
+  if (arr.options.ambiLight === true) {
+    return true
+  }
+})
+
+console.log(hasAmbiLight)
+
+inventory.sort((a, b) => {
+  return  a.price - b.price
+})
+
+console.log(inventory)
+
+// OPDRACHT 2
+
+const soldTvsContainer = document.getElementById('sold-tvs');
+const boughtTvsContainer = document.getElementById('bought-tvs');
+const toBeSoldTvsContainer = document.getElementById('to-be-sold-tvs');
+
+const tvsSold = (arr) => {
+  let outcome = 0;
+  for (let i = 0; i < arr.length; i++) {
+    outcome += arr[i].sold;
+  } return outcome
+}
+
+const soldTvs = tvsSold(inventory);
+
+const tvsBought = (arr) => {
+  let outcome = 0;
+  for (let i = 0; i < arr.length; i++) {
+    outcome += arr[i].originalStock;
+  } return outcome
+}
+
+const boughtTvs = tvsBought(inventory)
+
+const toBeSold = () => {
+  return boughtTvs - soldTvs
+}
+
+soldTvsContainer.textContent = `Amount sold tv's = ${soldTvs}`
+boughtTvsContainer.textContent = `Amount bought tv's = ${boughtTvs}`
+toBeSoldTvsContainer.textContent = `Amount tv's to be sold = ${toBeSold()}`
+
+// OPDRACHT 3
+
+const tvBrands = inventory.map((tv) => {
+  return tv.brand
+})
+
+// console.log(tvBrands)
+
+function createList (arr) {
+  const tvBrandsContainer = document.getElementById('tv-brands-container');
+  arr.map((tv) => {
+    tvBrandsContainer.innerHTML += `<li>${tv.brand}</li>`
+  })
```

Comment:
Als je niets met de uitkomst van de map doet, dan kun je beter een forEach gebruiken
```suggestion
  arr.forEach((tv) => {
    tvBrandsContainer.innerHTML += `<li>${tv.brand}</li>`
  })
```